.github/workflows/tests_lkcpu_python.yml

```diff
@@ -258,7 +251,6 @@ jobs:
 
       - name: Test editable install
         run: |
-          rm -r build
```

Comment:
Not because the CI does not find the `build` dir :thinking: [CIs error](https://github.com/PennyLaneAI/pennylane-lightning/actions/runs/11707796453/job/32608339793)

Reply:
one question. why don't we use make commands on CIs?

Reply:
To be sure to remove all the build dir. Could be `rm -rf build_*`? :thinking:

Reply:
`build_*` only includes the py files while both can be removed but the target should be `build` as it includes both arch `bdist` and `libs`.

Reply:
Ok, I agree that we don't need this step as the editable installation triggers before the followup Lightning installations.